src/main/java/io/strimzi/kafka/bridge/config/BridgeConfig.java

```diff
@@ -84,4 +86,8 @@ public String toString() {
                 ",httpConfig=" + this.httpConfig +
                 ")";
     }
+
+    public String getBridgeID() {
```

Comment:
Why are we calling it "bridge id" when the corresponding key in the consumer JSON and in the `BridgeEndpoint` is called "name"? And the configured value is more of a prefix than a complete id/name, so the naming ought to reflect that, no?

Reply:
The "name" is for the endpoint which ends up in the consumer name. Actually, it's a bridge id/name but used as prefix for consumer name creation.

Reply:
So it's just "identifying" a consumer, not the bridge itself?

Reply:
it identifies the bridge but is used as prefix for consumer names.

Reply:
But that's kind of my point: How does this "identify the bridge"? AFAICS it is _only_ used as a prefix for consumer names. Thus is only indirectly identified the bridge and primarily identifies the consumers. Therefore calling it `consumerIdPrefix` is a lot closer to the truth of what this is used for (and thus more easily understood by users). Do you have plans for being able to identify the bridge in some other way, which will make the "bridge id" more meaningful?

Reply:
I don't have right now, but it could be used in the future in that way. If we call it "consumerIdPrefix" today we are going to do more changes in the future if the bridge id could be useful.
What's the problem saying that it's a bridge identifier (which will be the `metadata.name` from the K8S resource) and it's also used as prefix for consumer names. What's wrong with that? :-)